### DISCUSSION TITLE
Target Framework Monikers

### DISCUSSION THREAD

**Discussion by MichelMichels** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3203] [General]
After removing the .NET Core 3.1 moniker, I noticed some inconsistencies across the projects. I have put them in a table below.

| Project | `net462` | `net472` | `net6.0-windows` | `net7.0-windows` | `net7` |
| ------- | ----------| ----------|---------|----------| --- |
| Demos\MahAppsDragablzDemo |     | X  |  X  |  X  | |
| Demos\MateriallDesign3Demo |     | X  |  X  |  X  | |
| Demos\MaterialDesignDemo |     | X  |  X  |  X  ||
| Demos\MahAppsDragablzDemo |     | X  |  X  |  X  ||
| Tools\mdresgen |     |  |   |   | X |
| MaterialDesignColors.Wpf | X    |   |  X  |  X  ||
| MaterialDesignColors.Wpf.Tests |     | X  |  X  |  X  ||
| MaterialDesignThemes.MahApps |  X   |   |  X  |  X  ||
| MaterialDesignThemes.UITests |     |   |    |  X  ||
| MaterialDesignThemes.Wpf|   X  |   |  X  |  X  ||
| MaterialDesignThemes.Wpf.Tests|     | X  |  X  |  X  ||

Would there be a specific reason to support .NET Framework 4.6.2 and 4.7.2? This seems redundant and I think we should move to 4.6.2 for all projects as this is the minimal supported version of .NET Framework.

**Comment by MichelMichels** [comment]
Also, is there a reasoning for both supporting .NET 6.0 and 7.0? Wouldn't 7.0 cover everything?

**Reply by Keboo** [reply]
The reason because if someone is building an app targeting .NET6 we want to have that runtime supported. 
For all of the NuGet projects, the idea is we want to have versions for .NET Frameworks, .NET6, and .NET7 all packaged inside of the `lib` so that NuGet can pick the best one based on the developer's TFM for their app.
